SumoForkClass.php

```diff
@@ -117,7 +117,7 @@ public function initErrbit()
                     $this->debug = $this->getContainer()->hasParameter('fork.debug');
                 }
                 $this->errbitApiKey = $this->getContainer()->getParameter('sumo.errbit_api_key');
-            } catch (\Symfony\Component\DependencyInjection\Exception\ParameterNotFoundException $e) {
```

Comment:
Yes, but the `getParameter()` function throws a `InvalidArgumentException` https://github.com/symfony/dependency-injection/blob/master/ContainerInterface.php#L78